buying_selling/posts/api/views.py

```diff
@@ -58,10 +58,12 @@ def retrieve(self, request, pk, format=None):
         post_serializer = PostDetailSerializer(post, context={'request': request}).data
         image_serializer = ImageSerializer(images, many=True, context={'request': request}).data
         i = 0
+        images = []
         print(image_serializer)
         for image in image_serializer:
-            post_serializer[f"image{i}"] = image
+            images.append(image)
             i += 1
+        post_serializer['images'] = images
```

Comment:
@Saurav-Shrivastav  You can remove `i` if it's not being used

Reply:
Done